GoToProject/src/containers/Reactions/styled.js

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components/native';
+import {colors} from '../../library/constants/colors';
+import {TitleText, FlexContainer} from '../../components/styled';
+export const Cart = styled(FlexContainer)`
+  border-radius: 8px;
+`;
+export const Starts = styled.View`
+  background: #59bcf4;
```

Comment:
You can import it from your color library.

Reply:
done